src/views/collection-view/index.tsx

```diff
@@ -55,6 +56,7 @@ function CollectionView({
 					formatTutorialCard(t, collection)
 				)}
 			/>
+			{layoutProps.isCertificationPage && <CertNewsletterSignup />}
```

Comment:
```suggestion
			{layoutProps.isCertificationPage && (
				<>
					<div className={s.signUpForm}>
						<SignupFormArea />
					</div>
				</>
			)}
```

Reply:
could also add a `className` prop to `<SignupFormArea />` to cut down on the div soup